packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -19,7 +19,7 @@ export interface EnqueueLinksOptions {
     limit?: number;
 
     /** An array of URLs to enqueue. */
-    urls: string[];
+    urls?: string[];
```

Comment:
This is not optional technically... It *is* for context aware equeueLink, but not for the one imported from core

Reply:
i know, but i will much rather have it this way, without 4 extra interfaces and all the type jugling. the runtime validation is there, this is just about types, and this implementation is muuuuch simpler to understand as well as maintain.